sympy/functions/special/polynomials.py

```diff
@@ -831,15 +855,18 @@ def _eval_rewrite_as_polynomial(self, n, x, **kwargs):
 
 class assoc_legendre(Function):
     r"""
-    assoc_legendre(n,m, x) gives :math:`P_n^m(x)`, where n and m are
+    ``assoc_legendre`` (n, m, x) gives $P_n^m(x)$, where n and m are
```

Comment:
The part in parentheses should be in code too, as its part of the code. `assoc_legendre(n, m, x)`